DataDog/terraform-provider-datadog
# [datadog_powerpack] set reasonable default group widget layout value

Shouout to @nkzou  for finding this bug - adding a terraformed powerpack onto a dashboard causes it to disappear on refresh. This is happening because the API accepts powerpack inputs without a `group_widget` layout property, which breaks on dashboards.

To fix this, I am adding a reasonable default value for the powerpack group_widget layout.

To test:

1. Create a powerpack TF resource

```
resource "datadog_powerpack" "foo2" {
  name = "Sample Powerpack"
  tags = ["tag:foo1", "tag:foo2"]
  description = "Test Powerpack 2"

  template_variables {
    defaults = ["defaults"]
    name     = "datacenter"
  }
  widget {
    note_definition {
      content          = "note widget"
      background_color = "red"
    }
  }
}
```
3. Add powerpack resource to dashboard. 
4. Refresh dashboard. It should still be there.
